mbuild/tests/test_compound.py

```diff
@@ -837,3 +840,15 @@ def test_to_networkx_names_only(self):
         assert graph.number_of_nodes() == 9
 
         assert all([isinstance(n, str) for n in graph.nodes()])
+
+    def test_from_trajectory(self):
+        comp = mb.Compound()
+        traj = mdtraj.load(get_fn('spc.pdb'))
+        comp.from_trajectory(traj)
+        assert comp.children[0].name == 'SPC'
+
+    def test_from_parmed(self):
```

Comment:
Could we also have a test(s) that loads in a more complex file, a.k.a. one with multiple residues, and make sure the hierarchy is still kosher? Maybe as simple as something with a single water and singe ethane in a box.